composer.json

```diff
@@ -5,7 +5,7 @@
     "spryker/guzzle": "^2.1.0"
   },
   "autoload": {
-    "psr-0": {
+    "psr-4": {
```

Comment:
Are you sure you can just switch without changing the namespace value?
Does that work?